enterprise/server/remote_execution/filecache/filecache.go

```diff
@@ -41,6 +42,8 @@ const (
 	tmpDir = "_tmp"
 )
 
+var enableAlwaysClone = flag.Bool("executor.enable_always_clone_filecache", false, "If true, files from the filecache will always be cloned instead of hardlinked")
```

Comment:
The filecache flag names are getting wild :face_with_spiral_eyes: 
- `executor.local_cache_{directory,size_bytes}`
- `executor.enable_fastcopy_reflinking`
- `executor.enable_always_clone_filecache`

the ship has prob sailed on `enable_fastcopy_reflinking` but maybe instead of `enable_always_clone_filecache` we can call it `local_cache_always_clone`?

Reply:
Done

Reply:
I think we can clean this up with flag.Alias